torch/linalg/__init__.py

```diff
@@ -378,6 +378,73 @@
             [1, 2, 2, 2]])
 """)
 
+vector_norm = _add_docstr(_linalg.linalg_vector_norm, r"""
+linalg.vector_norm(input, ord=None, keepdim=False, *, out=None, dtype=None) -> Tensor
+
+Returns the vector norm of a given tensor. 
+
+Args:
+    input (Tensor): The input tensor. If the input has more than one dimension,
+        it is interpretted as a tensor flattened to one dimension.  The input
```

Comment:
"The input data type must be either a floating point or complex type." 

This is good but we should try to consistently use "Supports..." statements like torch.sub() does (see https://pytorch.org/docs/master/generated/torch.sub.html?highlight=sub#torch.sub) and enumerate the supported dtypes there at the end of the description.